elements/pfe-content-set/src/pfe-content-set.js

```diff
@@ -24,9 +24,16 @@ class PfeContentSet extends PFElement {
   }
 
   get isTab() {
+    var breakpointValue;
+    if (this.hasAttribute('pfe-breakpoint')) {
```

Comment:
Can you update the single quotes to double for consistency?

Reply:
Related: prettier should be running on commit, but it doesn't seem to be.  I'll look into it.

Reply:
updated.

Reply:
Related: Attempting to fix prettier in #726.